kuma/wiki/views/document.py

```diff
@@ -1005,3 +1006,28 @@ def _document_api_PUT(request, document_slug, document_locale):
         resp.status_code = 205
 
     return resp
+
+
+@process_document_path
+@waffle_flag('bc-signals')
+def bc_signal(request, document_slug, document_locale):
+    """Handle BC signals."""
+
+    doc, _ = _get_doc_and_fallback_reason(
+        document_locale, document_slug
+    )
+
+    if doc is None:
+        raise Http404
+
+    if request.is_ajax():
+
+        user = request.user if request.user.is_authenticated else None
+
+        BCSignal.objects.create(
+            document=doc,
+            user=user
+        )
+
+        return HttpResponse()
```

Comment:
I assume the response is empty. Perhaps a 204 response would be most appropriate?